src/sentry/consumers/__init__.py

```diff
@@ -160,6 +185,7 @@ def ingest_replay_recordings_buffered_options() -> List[click.Option]:
     "ingest-monitors": {
         "topic": settings.KAFKA_INGEST_MONITORS,
         "strategy_factory": "sentry.monitors.consumers.monitor_consumer.StoreMonitorCheckInStrategyFactory",
+        "click_options": ingest_monitors_options(),
```

Comment:
@untitaker The way this `KAFKA_CONSUMERS` stuff was implemented made it super easy to pass this through as options. Thanks for writing it like this 💯